runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -280,6 +289,25 @@ public void onConfigChanged(final ConfigChanged event)
 					panel.setFilterMode(config.subscriptionFilter());
 					updateList();
 					break;
+				case "customWorldCycle":
+					setCustomWorldCycle(event.getNewValue());
+					break;
+			}
+		}
+	}
+
+	private void setCustomWorldCycle(String worldList)
```

Comment:
What is `worldList` used for? It seems to me this function always receives `config.customWorldCycle()` as this value and it's not needed.

Reply:
The parameter was passed under the assumption that the event fired prior to config receiving the change not after; As this isn't the case removed as redundant.